### ISSUE TITLE
Stop-gap solution that doesn't drop time in timestamps 

### ISSUE THREAD

**Issue by iingram** [conservationtechlab/otolith#21]
An issue has been discovered with timestamps losing time during SD card writes.  I'll write the details of this up more in a separate issue.  However, in the meantime we decided we would implement a basic code that does not involve interrupts at all but just has the sample accumulation and SD card writing in the main Arduino loop and tolerates gaps in polling of IMU data (kept as far apart as usual) but (hopefully) does not have any timestamp drift.  So this current issue is about making that stop-gap solution so we definitely have something to run on this unit next week. Once this is complete we can return to implementation of solution that does not have SD card write gaps.
